spec/index.html

```diff
@@ -1578,7 +1578,17 @@ <h3>Parsing Turtle in HTML</h3>
 
 <section id="privacy-considerations" class="appendix informative">
   <h2>Privacy Considerations</h2>
-  <p class="ednote">TODO</p>  
+  <p>Turtle is used to express arbitrary application data,
```

Comment:
This change might be considered for consistency with N-Quads, N-Triples, etc., even though `Turtle` is a singular lexical name.
```suggestion
  <p>The Turtle format is used to express arbitrary application data,
```